src/components/Dashboard/DashboardComponent.jsx

```diff
@@ -107,27 +103,42 @@ class DashboardComponent extends Component {
               </Grid.Row>
             </Grid>
           </div>
-          <div id="art-charts">
-            <Grid>
-              <Grid.Row columns={2}>
-                <Grid.Column>
-                  <div id="area-chart">
-                    <Chart1 />
-                  </div>
-                </Grid.Column>
-                <Grid.Column>
-                  <Grid id="right-chart-grid">
-                    <Grid.Row columns={2} style={{ border: 'solid 1px #707070', borderRadius: '10px' }}>
-                      <Grid.Column>
-                        <Chart2 />
-                      </Grid.Column>
-                      <Grid.Column>
-                        <Chart3 />
-                      </Grid.Column>
-                    </Grid.Row>
-                  </Grid>
-                </Grid.Column>
-              </Grid.Row>
+
+          <Divider />
+
+          <div id="asset-links-section">
+            <Grid columns={4} stackable>
+              <Grid.Column>
+                <Link to="/assets">Asset List</Link>
```

Comment:
This should not be here. It should remain in the navigation bar with a title of `Assets`. However, it should not have the dropdown